epochStart/bootstrap/startInEpochScheduled.go

```diff
@@ -355,8 +364,12 @@ func (ses *startInEpochWithScheduledDataSyncer) getScheduledTransactionHashes(he
 
 	scheduledTxs := make(map[string]struct{})
 	for _, mb := range miniBlocks {
+		if mb.GetReceiverShardID() != header.GetShardID() {
+			continue
+		}
 		for _, txHash := range mb.TxHashes {
 			scheduledTxs[string(txHash)] = struct{}{}
+			log.Debug("startInEpochWithScheduledDataSyncer.getScheduledTransactionHashesWithDestMe", "hash", txHash)
```

Comment:
this is a little too much log.debug - please delete

Reply:
There is only for scheduled txs, if they exist, from one single block at bootstrap -> start in epoch. So, it is printed only once, but it helps for debugging if needed!

Reply:
ah. I missed that it is only start in epoch. alright.